aspenTools/python/source/aspen/blender/modeling/modeling_pipeline/panels.py

```
@@ -10,4 +10,8 @@ class ASPENMODELING_PT_panel(bpy.types.Panel):
     bl_options = {'DEFAULT_CLOSED'}
 
     def draw(self, context):
-        layout = self.layout
\ No newline at end of file
+        layout = self.layout
+
+        row = layout.row()
```

Ruff (0.14.4)

Local variable `row` is assigned to but never used

Remove assignment to unused variable `row`

(F841)